submissions/israelOladosu/php-db/classes/queryDb.class.php

```diff
@@ -30,4 +30,27 @@ public function getAllCustomers()
         }
     }
 
+    public function getOrders($id)
+    {
+        try {
+
+            $sql = "SELECT items.quantity, items.unit_price,
+            items.total_amount, orders.made_at, orders.id, 
+            orders.delivery_address, products.product_name 
```

Comment:
I don't think there is a column called `delivery_address` on orders table. I think the customer location should be coming from the `addresses` table

Reply:
oh! I actually altered the orders table to have a delivery_address column forgetting totally that I had a table for addresses. I have made the necessary adjustment now. I will also add the new screenshots

Reply:
Added the new screenshot now. Thank you.